ocaml/xapi-client/tasks.ml

```diff
@@ -23,7 +23,7 @@ module TaskSet = Set.Make (struct
 end)
 
 (* Return once none of the tasks have a `pending status. *)
-let wait_for_all_inner ~rpc ~session_id ?all_timeout ~tasks =
+let wait_for_all_inner ~rpc ~session_id ~all_timeout ~tasks =
```

Comment:
adding a `()` at the end might've been another way to fix it

Reply:
Indeed, I thought making the `option` explicit rather than implicit would be a good thing for readability